src/main/java/module-info.java

```diff
@@ -13,6 +14,9 @@
 import org.cryptomator.common.locationpresets.OneDriveMacLocationPresetsProvider;
 import org.cryptomator.common.locationpresets.OneDriveWindowsLocationPresetsProvider;
 import org.cryptomator.common.locationpresets.PCloudLocationPresetsProvider;
+import org.cryptomator.common.integrations.sslcontext.SSLContextWithMacKeychain;
+import org.cryptomator.common.integrations.sslcontext.SSLContextProvider;
+import org.cryptomator.common.integrations.sslcontext.SSLContextWithWindowsCertStore;
```

Comment:
drop the `integrations` part from the package name, as this is rather something internal. I'd suggest `org.cryptomator.networking`

Reply:
Changed in https://github.com/cryptomator/cryptomator/pull/3675/commits/8d93abb37e6d0272ed2285bbfb493e582de459d8